cmd/git-sync/main.go

```diff
@@ -695,3 +691,27 @@ func setupGitCookieFile() error {
 
 	return nil
 }
+
+func triggerWebhook(ctx context.Context, ch chan webhookRepoInfo, branch, rev, gitRoot string) error {
+	info := webhookRepoInfo{
+		Branch: branch,
```

Comment:
do we need branch or just hash?  Hash should be unique enough for anything.  Branch and tag are symbolic references

Reply:
I agree. Hash should be enough.